style.css

```diff
@@ -1,3 +1,126 @@
+*,
+body {
```

Comment:
I don't have too much to say about your CSS, it's getting over my head now lol, but I did want to a) say it's good that you did responsive design and b) note that the color of the text and the formatting of the cards changes between breakpoints which isn't really good practice